//python -m grpc_tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/main.proto

syntax = "proto3";

service TestService{

	rpc testMethod(TestRequest) returns (TestResponse){}
	rpc testMethod2(TestRequest) returns (TestResponse){}	
}

service TestService2{

	rpc testMethod(TestRequest) returns (TestResponse){}
	rpc testMethod2(TestRequest) returns (TestResponse){}	
}



message TestRequest{
	int32 number = 1;
	string text = 2;
}


message TestResponse{
	int32 number = 1;
	string text = 2;
}
